views/windows.tmpl

```diff
@@ -0,0 +1,145 @@
+{{define "windows"}}
+<!DOCTYPE html>
+<html lang="en">
+
+{{template "html-head" "Decred Blocks Windows"}}
+<body class="{{ theme }}">
+    {{template "navbar" . }}
+    <div class="container main" data-controller="main">
+        <h4>Windows</h4><h6>A window is a range of {{$.WindowSize}} blocks or approximately 12hrs. After each window mining; difficulty and ticket price are adjusted.</h6>
```

Comment:
This page is for PoS price windows, not PoW, correct?  The work mining diff window is set separately (`WorkDiffWindowSize`)

Reply:
There are two variables, but for all 4 net types they are set to identical values. Both sdiff and diff value can be shown on the windows list.

Reply:
Hopefully they are never changed.